src/esp/bindings/SimBindings.cpp

```diff
@@ -84,97 +84,169 @@ void initSimBindings(py::module& m) {
+          R"(Get the Simulator's ObjectAttributesManager instance for configuring object templates.)")
+      .def(
+          "get_physics_template_manager",
+          &Simulator::getPhysicsAttributesManager,
+          pybind11::return_value_policy::reference,
+          R"(Get the Simulator's PhysicsAttributesManager instance for configuring PhysicsManager templates.)")
+      .def(
+          "get_stage_template_manager", &Simulator::getStageAttributesManager,
+          pybind11::return_value_policy::reference,
+          R"(Get the Simulator's StageAttributesManager instance for configuring simulation stage templates.)")
+      .def(
+          "get_physics_simulation_library",
+          &Simulator::getPhysicsSimulationLibrary,
+          R"(Query the physics library implementation currently configured by this Simulator instance.)")
       /* --- Object instancing and access --- */
-      .def("add_object", &Simulator::addObject, "object_lib_index"_a,
-           "attachment_node"_a = nullptr,
-           "light_setup_key"_a = assets::ResourceManager::DEFAULT_LIGHTING_KEY,
-           "scene_id"_a = 0)
-      .def("add_object_by_handle", &Simulator::addObjectByHandle,
-           "object_lib_handle"_a, "attachment_node"_a = nullptr,
-           "light_setup_key"_a = assets::ResourceManager::DEFAULT_LIGHTING_KEY,
-           "scene_id"_a = 0)
+      .def(
+          "add_object", &Simulator::addObject, "object_lib_id"_a,
+          "attachment_node"_a = nullptr,
+          "light_setup_key"_a = assets::ResourceManager::DEFAULT_LIGHTING_KEY,
+          "scene_id"_a = 0,
+          R"(Instance an object into the scene via a template referenced by library id. Optionally attach the object to an existing SceneNode and assign its initial LightSetup key.)")
+      .def(
+          "add_object_by_handle", &Simulator::addObjectByHandle,
+          "object_lib_handle"_a, "attachment_node"_a = nullptr,
+          "light_setup_key"_a = assets::ResourceManager::DEFAULT_LIGHTING_KEY,
+          "scene_id"_a = 0,
+          R"(Instance an object into the scene via a template referenced by its handle. Optionally attach the object to an existing SceneNode and assign its initial LightSetup key.)")
       .def("remove_object", &Simulator::removeObject, "object_id"_a,
            "delete_object_node"_a = true, "delete_visual_node"_a = true,
-           "scene_id"_a = 0)
-      .def("get_object_initialization_template",
-           &Simulator::getObjectInitializationTemplate, "object_id"_a,
-           "scene_id"_a = 0)
+           "scene_id"_a = 0, R"(
+        Remove an object instance from the scene. Optionally leave its root SceneNode and visual SceneNode on the SceneGraph.
+
+        .. note-warning::
+
+            Removing an object which was attached to the SceneNode
+            of an Agent expected to continue producing observations
+            will likely result in errors if delete_object_node is true.
+            )")
```

Comment:
@mosra Not sure why this isn't generating a method detail block with a warning note. I tried copying the white-space formatting for the `semantic_scene` (ln 62). Any thoughts?

Reply:
Any warning printed (if you can see it among all the others)? What does it generate instead?

Reply:
You can try tools like rstcheck to further validate it / find the specific error.

Reply:
It generates the brief doc, but no entry in the method Documentation section. I can't find any warnings about this either.
![image](https://user-images.githubusercontent.com/1445143/91916846-035c5b80-ec73-11ea-88d8-d64dacbbc905.png)

Reply:
You may need to un-indent things.  I am not sure how the rst parser feels about all that whitespace.